src/main/resources/application.yml

```diff
@@ -11,6 +11,11 @@ spring:
             username: "\${MYPAY_DB_USER}"
             password: "\${MYPAY_DB_PASSWORD}"
             driverClassName: "org.postgresql.Driver"
+        mypivot:
+            jdbc-url: "jdbc:postgresql://\${MYPAY_DB_HOST}:5432/mypivot"
+            username: "\${MYPAY_DB_USER}"
+            password: "\${MYPAY_DB_PASSWORD}"
```

Comment:
use different environment variable names here!
The use of the same database server is a deployment detail: we will use the same values inside the values.yml

Reply:
done